resources/tokens/z-index.yml

```diff
@@ -8,6 +8,10 @@ props:
     value: "9999"
   z-index-sticky:
     value: "100"
+  z-index-docked:
+    value: "4"
```

Comment:
This leaves some room to wiggle with z-indexes between docked and default in future. It could have been 10 or 50 too. We could start with 1 as default and 2 for docked too.

Reply:
I think we should decide and document the z-indexing and try to stick to it. Otherwise it may be difficult to deal with in the future. Using increments of 10's for example gives more information about ordering and "importance" at a glance than random inrements. At lest I feel that 10, 30, 50, 100, 250... is somehow a bit easier to understand with a glance than 3,7, 34, 68, 75, 767... even if they are not ordered while you look at them.